hadoop-hdds/docs/content/feature/HA.zh.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "高可用"
```

Comment:
This should be set to `特点`, otherwise this page will not be displayed on the left menu.
![image](https://user-images.githubusercontent.com/14295594/99340337-c2afce00-28c2-11eb-8a6a-c67e82e1d066.png)

Reply:
Thanks @yuyang733 for the reply.

`特性` --> `特点`  -> Yes, I marked the wrong code block, it should be here.